loadtest/main.go

```diff
@@ -135,6 +151,36 @@ func (c *LoadTestClient) generateMessage(config Config, key cryptotypes.PrivKey,
 				Amount: sdk.NewInt(1),
 			}),
 		}
+	case DistributeRewards:
+		adminKey := c.SignerClient.GetAdminKey()
+		msg = &banktypes.MsgSend{
+			FromAddress: sdk.AccAddress(adminKey.PubKey().Address()).String(),
+			ToAddress:   sdk.AccAddress(key.PubKey().Address()).String(),
+			Amount: sdk.NewCoins(sdk.Coin{
+				Denom:  "usei",
+				Amount: sdk.NewInt(10000000),
+			}),
+		}
```

Comment:
are we only distributing to 1 validator?

Reply:
yeah, plan is to delegate more from the node_admin account from node-0 so that it can collect the rewards from other validators